Resources/doc/filters/general.rst

```diff
@@ -0,0 +1,204 @@
+
+.. default-role:: code
+
+General Filters
+===============
+
+Background
+----------
+
+.. _filter-background:
+
+The built-in `background` filter performs layer transformations
+(which includes creating and mergin layer operations). This
+filter exposes a number of `background options`_ which may be used
+to configure its behavior.
+
+Example configuration::
+
+    # app/config/config.yml
+
+    liip_imagine:
+        filter_sets:
+
+            # name our filter set "my_background_filter"
+            my_background_filter:
+                filters:
+
+                    # use and setup the "background" filter
+                    background:
+
+                        # set the background color to #00ffff
+                        color: '#00ffff'
+
+                        # set a size different from the input image
+                        size: [1026, 684]
+
+                        # center input image on the newly created background
+                        position: center
+
+
+.. note::
+
+    The background color is only visible through transparent image sections (if
+    any), unless a **size** option is provided, in which case a new image is
+    created and the input image is placed on top according to the **position** option.
+
+
+Background Options
```

Comment:
Changed to using `.. default-role:: code` globally, enabling single backticks (facilitating easy RST -> MD). Please advise if you think this is bad practice or otherwise think it should not be handled like this. Thanks again for the review!